setup.cfg

```diff
@@ -49,3 +49,8 @@ disallow_untyped_decorators = true
 disallow_any_generics = true
 check_untyped_defs = true
 allow_redefinition = true
```

Comment:
Can we remove this globally, please? Unfortunately, I missed the PR where this made it in, otherwise I would have raised it

Reply:
This is only for the plugins file, it is not global. If I remember correctly you were happy with it.

Reply:
See my comment [here](https://github.com/beetbox/beets/pull/5738#issuecomment-2909819896)

Reply:
Removed for the metadata_plugins section, which is the relevant one for this commit imo.

```toml
[[mypy-beets.metadata_plugins]]
disallow_untyped_decorators = true
disallow_any_generics = true
check_untyped_defs = true
```

Reply:
Yeah I have seen this. We can work around the redefinition, it is just a bit annoying and might require additional variables in some cases, but I see the benefits for iterators. 

> without making any changes

To get you right, you want me to remove the metadata_plugin section again? I think going for this incremental opt in approach is quite nice for adaption.